enable/constraints_container.py

```diff
@@ -277,7 +277,7 @@ def __components_changed(self, new):
         # Check the new components
         self._check_and_add_components(new)
 
-    def _component_size_hint_changed(self):
+    def _component_size_hint_changed(self, event):
```

Comment:
You should rename this method so that it doesn't look like a static notification handler.